packages/scene-composer/src/layouts/SceneLayout/components/FoldableContainer.scss

```diff
@@ -0,0 +1,26 @@
+// @use '~/@awsui-design-tokens/polaris.scss';
```

Comment:
should be `@use '@awsui/design-tokens/index.scss' as awsui;`
then use in code like `awsui.$color-border-divider-default`